bookshelf/test/e2e/BookshelfTest.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/*
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Samples\Bookshelf;
+
+use Behat\Mink\Driver\GoutteDriver;
+use Behat\Mink\Session;
+
+/**
+ * Class BookshelfTest
+ */
+class BookshelfTest extends \PHPUnit_Framework_TestCase
+{
+    use E2EDeploymentTrait;
+
+    const STEP = 'bookshelf';
```

Comment:
Just a heads up, this was a placeholder until I added directory support. So I'll refactor this out once we merge it.